### DISCUSSION TITLE
Support multiple model formats per version

### DISCUSSION THREAD

**Discussion by hassan-sd** [civitai/civitai#62] [Ideas]
Thank you for implementing the additional support for safetensors versions.

This new feature request is to allow multiple types **per version**.

As an example, I released HassanBlend1.4 in both cktp pickle format and safetensor format, allowing both uploads to live consecutively within the same version is ideal. 

Users can then decide which version they choose to download. Same for multiple CKTP or multiple Safetensor versions such as:

- HassanBlend 1.4.safetensor
- HassanBlend 1.4-pruned.safetensor
- HassanBlend 1.4.cktp
- hassanBlend 1.4-pruned.cktp 
etc

**Comment by JustMaier** [comment]
On the top of our list. Along with the ability to automatically convert it to either format for users.

**Comment by JustMaier** [comment]
@manuelurenah is going to be working on this one. Here's [the plan](https://hackmd.io/LcRCHTGCQumtChZttu8DqA?edit)

**Comment by JustMaier** [comment]
**This is live!**
![Demo of multiple files](https://user-images.githubusercontent.com/607609/208222159-22efb0c2-d23a-4a12-aa67-deb02ee8de26.png)

Now you can upload multiple files of the various types needed to support models. This allows you to upload pruned, full, and safetensor and ckpt versions of models.
